tools/PI/DevHome.PI/ViewModels/WinLogsPageViewModel.cs

```diff
@@ -65,9 +72,13 @@ public WinLogsPageViewModel()
 
         _insightsService = Application.Current.GetService<PIInsightsService>();
 
+        _filterMessageText = string.Empty;
         _winLogEntries = [];
         _winLogsOutput = [];
         _winLogsOutput.CollectionChanged += WinLogsOutput_CollectionChanged;
+        _winLogsView = new AdvancedCollectionView(_winLogEntries, true);
```

Comment:
>AdvancedCollectionView [](http://example.com/codeflow?start=27&length=22)

Looks like I should look into this for the WER page as well... looks like it does all the heavy lifting!

Reply:
Yes, it is a great option. However, was difficult to find it.